client-spark/spark-3/src/main/scala/org/apache/spark/shuffle/celeborn/CelebornShuffleReader.scala

```diff
@@ -89,6 +92,23 @@ class CelebornShuffleReader[K, C](
       }
     }
 
+    val exceptionMaker = new ExceptionMaker() {
+      override def makeException(
+          appShuffleId: Int,
+          shuffleId: Int,
+          partitionId: Int,
+          e: Exception): Exception = {
+        new FetchFailedException(
+          null,
+          handle.shuffleId,
+          -1,
+          -1,
+          partitionId,
+          SparkUtils.FETCH_FAILURE_ERROR_MSG + shuffleId,
```

Comment:
Include both appShuffleId and shuffleId in message (if they are different) to help debugging ?